### ISSUE TITLE
Issues with Steam

### ISSUE THREAD

**Issue by Sawtaytoes** [sandboxie-plus/Sandboxie#1157]
**Is your feature request related to a problem? Please describe.**
I'm using Sandboxie to install multiple copies of Steam. Is there a way to create a light-weight wrapper around an app where Sandboxie isn't as concerned about security? I saw a lot of settings like this in the app, but some games fail to load with Sandboxie because it's still pretty restrictive.

Ideally, I'd only sandbox Steam and not the games.

**Describe the solution you'd like**
A simple setting to modify the install and runtime directory, but nothing else. That or a way to sandbox only the Steam executable, but not the games it runs (since they're already gonna run from the sandbox directory).

**Describe alternatives you've considered**
I've tried making Sandboxie more open and less locked-down, but it's still there trying to control everything about the app. I also looked at the Steam executable to see if I could set it up to allow multiple running instances, but my skillset is n00b, and I couldn't figure out a way to avoid using Sandboxie.

**Additional context**
Steam allows running only one instance. No matter where it's installed, even if you rename the executable to `steam2.exe`, when opening it up, it makes sure there's only 1 copy of itself running. Who knows what could happen if there were multiple competing registry entries. Maybe that's what's happening now.

This poses a problem because Sandboxie comes with many limitations and quite a bit of overhead. All I want is for it to fake the directory bits and app ID, but I'm assuming there's a lot more going on behind the scenes.

**Comment by DavidXanatos** [comment]
have you tried the App Compartment mode
and the BreakoutProcess feature?

that combination should be enough for your use case, please let me know if that is so

**Comment by isaak654** [comment]
> I went ahead and added BreakoutProcess=y to the config. Is that what you're saying? 
 
No, it isn't correct. See [BreakoutProcess](https://sandboxie-plus.github.io/sandboxie-docs/Content/BreakoutProcess.html) and [BreakoutFolder](https://sandboxie-plus.github.io/sandboxie-docs/Content/BreakoutFolder.html) documentation.

> Is there a place I can see this in the app?

It should be added graphically in a future Plus version.

**Comment by Sawtaytoes** [comment]
Hmm... Based on the description, I don't think `BreakoutProcess` is what I'm wanting.
> BreakoutProcess is a sandbox setting in [Sandboxie Ini](https://sandboxie-plus.github.io/sandboxie-docs/Content/SandboxieIni.html) available since v1.0.8 / 5.55.8. **It specifies which applications shall run unsandboxed when launched within the sandbox.** A combination of this and ForceProcess allows for a simple priority system.

I need Steam to be sandboxed to the point where I can open 2 instances at the same time; the rest I don't care about right now. Does `BreakoutProcess` completely remove the sandbox too?

I also noticed the date when I originally posted this question. I believe App Container wasn't a thing back then. I did purchase Sandboxie since and have already enabled App Container, but I haven't done as much testing with it since that purchase. I wish I noted which games didn't work, so I could easily re-test.
